src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -26,14 +26,6 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
-    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
-  },
-  {
```

Comment:
Actually I meant to add `noSuggestion: true,`, showing the actual casts vs max casts in the cast efficiency would still be useful towards reminding people to use the ability. Especially on hard progress, using defensives often helps.